files/systemd/mediaindexer.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=Media indexing service
 Requires=ls-hubd_private.service ls-hubd_public.service
```

Comment:
@tofee @nizovn Not sure we need a requires here too? I guess not, but I'm not too familiar with systemd